Support/Testing/Travis/before-install.py

```diff
@@ -10,12 +10,18 @@
 ##
 
 import os
+import sys
 from subprocess import check_call
 
 repositories = []
 keys = []
 
 target = os.getenv('TARGET')
+travis_os = os.getenv('TRAVIS_OS_NAME')
```

Comment:
Same here, you don't need `travis_os`, you can just use `target` and do `if target == 'Darwin':`.